me.aandrew.ytdownloader.yml

```diff
@@ -0,0 +1,63 @@
+app-id: me.aandrew.ytdownloader
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
```

Comment:
I would reconsider changing this to a specific directory but it's not a blocker.